LFGBulletinBoard/LibGPIMinimapButton.lua

```diff
@@ -155,12 +192,14 @@ local MinimapShapes = {
 	["TRICORNER-BOTTOMRIGHT"] 	= {false, true, true, true},
 }
 
+-- Updates the position of the minimap button based on the calculated distance and angle.
 function MinimapButton.UpdatePosition()
-	local w = ((Minimap:GetWidth() / 2) + 10) * MinimapButton.db.distance
-	local h = ((Minimap:GetHeight() / 2) + 10) * MinimapButton.db.distance
-	--local r=math.rad(MinimapButton.db.position)
-	--MinimapButton.button:SetPoint("CENTER", Minimap, "CENTER", w * math.cos(r), h * math.sin(r))
-	local rounding=10
+	if MinimapButton.isUsingLibDBIcon then --[[position updates handled by LibDBIcon]] return end;
+
+	local radiusOffset = LibDBIcon and LibDBIcon.radius or 10 -- better align with any LibDBIcons.
```

Comment:
if `LibDBIcon` is available and the user is not using it to manage the minimap button, then we should still try to match thier `radius` value to better align. Otherwise we can fallback to `10` (which resulting on the button sitting on the edge instead of through it)

Reply:
Before (slightly higher than LibDBIcons):
![image](https://github.com/user-attachments/assets/c3fd1ccd-fbb9-40ad-8e74-c1c505a562bf)
After (inline):
![image](https://github.com/user-attachments/assets/a45ec932-2b98-42b1-8b80-70fe6d4951e0)